JS-counting/script.js

```diff
@@ -0,0 +1,35 @@
+console.log("Part 1");
+for (i = 1; i <= 10; i++) {
+    console.log(i);
+}
+
+for (i = 10; i >= 1; i--) {
+    console.log(i);
+}
+
+console.log("Part 2")
+let j = 1;
+while (j <= 10) {
+    console.log(j);
+    j++;
+}
+j = 10;
+while (j >= 1) {
+    console.log(j);
+    j--;
+}
+
+console.log("Part 3")
+let k = 1
+do {
+    console.log(k);
+    k++;
+} while (k <= 10);
+k = 10;
```

Comment:
Separate code blocks with a new line

```suggestion
} while (k <= 10);

k = 10;
```